### PR TITLE
Add optional onPaginationChange function prop

### PR DESCRIPTION
**Pull request by weskamm**
Calls the optional handler `onPaginationChange`, if given as prop

### CODE REVIEW

**Comment by hblitza** [comment]
Looks good!

File: src/component/FeatureInfoGrid/FeatureInfoGrid.tsx [hunk 1]
```diff
@@ -144,6 +146,9 @@ export class FeatureInfoGrid extends React.Component<FeatureInfoGridProps, Featu
       selectedFeat,
       currentPage: newIdx
     });
+    if (this.props.onPaginationChange) {
+        this.props.onPaginationChange(newIdx);
```
Comment (mholthausen) [inline src/component/FeatureInfoGrid/FeatureInfoGrid.tsx]:
```suggestion
      this.props.onPaginationChange(newIdx);
```

Comment (mholthausen) [reply src/component/FeatureInfoGrid/FeatureInfoGrid.tsx]: There seems to be a linting error

**Review by mholthausen — APPROVED** [review]
Linting problem got already solved, so: :+1:

**Comment by weskamm** [comment]
Thanks for reviews!

**Pull request merged**